packages/components/src/components/navbar/navbar.tsx

```diff
@@ -19,6 +19,7 @@ export class Navbar {
   @Prop() fixed: boolean = true;
   @Prop() showLogoAndAppname: boolean = true;
   @State() searchBarIsOpen: boolean = false;
+  @Prop({mutable: true}) logoHref: string = "";
```

Comment:
We don't mutate Props, because it is a bad practice.
Instead, create an **internalHref** State for internal usage. 
Please follow up our other examples of similar use-cases, such as the Button and Sidebar-item.